apis/certificatetemplateapi/certificatetemplateapi.yaml

```diff
@@ -0,0 +1,375 @@
+      parameters:
+      - in: path
+        name: templateId
+        type: string
+        required: true
+        description: 'Append a valid template ID To the Request URL'
+      - name: Content-Type
+        in: header
+        required: true
+        type: string
+        description: >-
+            The Content Type entity is the media type of the resource.Possible media types can be Application/json
+      - name: ts
+        in: header
+        required: true
+        type: string
+        description: 'The timestamp at which the **read certificate template** request was sent. '
+      - name: X-msgid
+        in: header
+        required: true
+        type: string
+        description: 'A unique ID that identifies the request, in case the same API is executed multiple times.'
+      - name: x-authenticated-user-token
+        in: header
+        required: true
+        type: string
+        description: 'The access token of the registered user sending the given API request. This is an alphanumeric string.'
+      - name: Authorization
+        in: header
+        required: true
+        type: string
+        description: 'To make use of the API, you require authorization. Raise a request to the administrator for the use of the API. You will receive the authorization key. Specify the key received, here.'
+      responses:
+        '200':
+          description: 'OK ! Operation successful."Read Certificate template" operation was executed successfully.'
+          schema:
+            allOf:
+             - $ref: 'https://raw.githubusercontent.com/project-sunbird/project-sunbird.github.io/dev/apis/definitions/envelope.yaml#/ApiResponse'
+             - $ref: '#/definitions/GetCertificateTemplateResponse'
+        '400':
+          description: 'BAD REQUEST. The "Read Certificate template" operation failed. Input for a mandatory parameter may be missing.'
+          schema:
+            $ref: '#/definitions/UpdateCertificateTemplateErrorResponse'
+        '500':
+          description: 'INTERNAL SERVER ERROR ! Looks like something went wrong. These errors are tracked automatically. Try refreshing the page. If the problem persists, contact info@sunbird.org.'          
+definitions:
+  AddCertificateTemplateRequest:
+    title: AddCertificateTemplateRequest
+    type: object
+    properties:
+      request:
+        $ref: '#/definitions/AddTemplateRequest'
+    required:
+    - request
+  AddTemplateRequest:
+    title: AddTemplateRequest
+    type: object
+    properties:
+      certificate:
+        description: certificate wrapper
+        type: object
+        properties:
+          template:
+            description: The template detail
+            type: object
+            properties:
+              identifier:
+                description: Teh pre-defined tempalte ID
+                type: string
+              name:
+                description: The name for the template
+                type: string
+              template:
```

Comment:
the 'template' field was chosen because this field was supposed to store the template zip itself, However currently the zip is uploaded to google drive and the url is provided.

Reply:
@rahul-tarento @maheshkumargangula There is a field called 'template' already in line 229. Can there be two fields with the same name?

Reply:
@Sunitashelar  'template' field has different levels, first 'template' on line 229 is part of 'certificate' on line 225. It holds complete detail of certificate template, However the 'template' on line 239 is part of certificate template which denotes the URL to the template zip file.